### DISCUSSION TITLE
Overseerr doesn't seem to work in Safari 14

### DISCUSSION THREAD

**Discussion by jby** [sct/overseerr#1071] [Help]
I never seem to get past the "loading" screen in Safari 14 on macOS 11 (Big Sur).

Is that expected?

<img width="2555" alt="Screenshot 2021-03-03 at 16 49 56" src="https://user-images.githubusercontent.com/1584888/109832436-9e44eb00-7c40-11eb-9677-37f12122a306.png">

It works fine in Firefox and Brave.

**Comment by TheCatLady** [comment]
Do you have any adblockers enabled in Safari?  It looks like something is blocking our requests to TMDb.

**Reply by jby** [reply]
I have AdGuard, but I've tried to disable it without any difference, besides it doesn't block the requests out of Sonarr/Radarr to TMDb...

**Comment by Xanadeath** [comment]


**Reply by jby** [reply]
Same for me, I hadn't tried it in iOS - but that works fine.

**Reply by samwiseg0** [reply]
I am not having an issue? Can you check the browser console for errors?

**Comment by samwiseg0** [comment]
I am not having an issue? Can you check the browser console for errors?

**Reply by jby** [reply]
Reloaded with console open:

<img width="1436" alt="image" src="https://user-images.githubusercontent.com/1584888/110202337-28bb6380-7e68-11eb-9c07-4d29856a0e6b.png">

**Reply by sct** [reply]
Check the network requests when you reload. I bet you there are a ton of failures in there.

**Reply by jby** [reply]
I have a bunch of rotating "waiting" - but no errors that I can see:

<img width="1432" alt="Screenshot 2021-03-06 at 21 19 22" src="https://user-images.githubusercontent.com/1584888/110219826-c1ca9880-7ec1-11eb-8ce8-870a01b1a0b1.png">

**Reply by nzrunner** [reply]
I am experiencing the same behaviour. My Docker console logs are littered with the following: -

`(node:250) UnhandledPromiseRejectionWarning: Error: [TMDb] Failed to fetch discover TV: getaddrinfo EAI_AGAIN api.themoviedb.org
    at TheMovieDb.getDiscoverTv (/app/dist/api/themoviedb/index.js:137:23)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /app/dist/routes/discover.js:225:18
(node:250) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 327)
(node:250) UnhandledPromiseRejectionWarning: Error: [TMDb] Failed to fetch discover movies: getaddrinfo EAI_AGAIN api.themoviedb.org
    at TheMovieDb.getDiscoverMovies (/app/dist/api/themoviedb/index.js:115:23)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /app/dist/routes/discover.js:36:18
(node:250) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 329)
2021-03-16T23:51:22.355Z [error][Movie]: [TMDb] Failed to fetch movie details: getaddrinfo EAI_AGAIN api.themoviedb.org 
2021-03-16T23:51:22.599Z [error][Movie]: [TMDb] Failed to fetch movie details: getaddrinfo EAI_AGAIN api.themoviedb.org 
2021-03-16T23:51:27.360Z [error][Movie]: [TMDb] Failed to fetch movie details: getaddrinfo EAI_AGAIN api.themoviedb.org 
2021-03-16T23:51:27.608Z [error][Movie]: [TMDb] Failed to fetch movie details: getaddrinfo EAI_AGAIN api.themoviedb.org`

I have added "api.themoviedb.org" to my Pi-Hole Whitelist and restarted the Pi-Hole. In addition, to be sure, I have restarted my router as well. Radarr can access TheMovieDB perfectly.

I note this behaviour has only started occurring since I upgraded Overseerr from 1.20.1 to 1.21.1

**Reply by jby** [reply]
All the XHR-requests are just hanging...
